tests/fs/readdir.test.ts

```diff
@@ -5,13 +5,17 @@ import { fs } from '../common.js';
 const testDir = 'test-dir';
 const testFiles = ['file1.txt', 'file2.txt', 'file3.txt'];
 const testDirectories = ['subdir1', 'subdir2'];
+const testSubdirFiles = ['file4.txt', 'file5.txt'];
 
 await fs.promises.mkdir(testDir);
 for (const file of testFiles) {
 	await fs.promises.writeFile(`${testDir}/${file}`, 'Sample content');
 }
 for (const dir of testDirectories) {
```

Comment:
If you only use `testDirectories` here, you can inline it.

Reply:
testDirectories was already being used.